app/src/main/res/values/integers.xml

```diff
@@ -7,4 +7,6 @@
 
     <!-- EditorInfo.IME_ACTION_* is defined by the framework. -->
     <integer name="ime_action_search">3</integer>
+
+    <integer name="webview_initial_zoom">100</integer>
```

Comment:
We have four definitions of this value but only two actually values: one is definitely unnecessary and I'm concerned the other one is redundant and perhaps unintuitive as well: can you consolidate?

Reply:
You're right, the device will pick the closest value that is equal to or lower than its own height, so we only need to define our ES5 and ES1 values.  ES2 will automatically fall into the E1 bucket.  But it feels strange to me that adjusting ES1 code will have repercussions on the ES2.

Maybe this just isn't a great use case for resources folders, and should be handled entirely differently.

Reply:
I initially also wanted a fallback number, but on second thought we should probably fail fast on this one.

Reply:
> Maybe this just isn't a great use case for resources folders, and should be handled entirely differently.

To use the resource folders comfortably, I think you have to think about it in terms of device characteristics instead of specific devices. In this case, all we're doing is increased the font zoom if the screen is below a certain size.

Reply:
> I initially also wanted a fallback number, but on second thought we should probably fail fast on this one.

fwiw, having 2 files where one is the default and the other is qualified – like we do with layouts – could be the right way to go. When thinking about resource folders in terms of device characteristics, it provides behavior to all devices without crashing. On the other hand, it isn't very explicit. 🤷‍♂

Reply:
I'm not even sure resource folders are appropriate for this.  They're designed for an unknown number of form factors with a crazy long tail.  Here, we have three devices, with maybe a handful more in the future.  It very much seems like the wrong tool for the job here, I'd much rather define a value for each device.  Not sure how I'd like to set that up, though.

Reply:
> It very much seems like the wrong tool for the job here, I'd much rather define a value for each device.

I'm not so sure: the resource system lets us bucket similar devices into groups, reducing the number of variants we have to explicitly define and, often, test. Instead of FFES 1, 2, 5, and potentially more, we can consider tiny, small, & normal devices (and hopefully we don't see even more obscure form factors like extra wide :P). Also, there are fewer duplicated values to remember to update or add.

For an example, let's say we have several integer values for each device:
- When a new device comes out, we'd need to redefine every value for the new device. This creates uncertainty, additional effort, & duplication: does the eng do their intuition to define these values or coordinate with UX? However, if the device matches our buckets, things might just work
- When adding a value, a developer needs to remember to consider all variants: using resources buckets, there are fewer variants to check

Reply:
I disagree (this feels like leaving a gaping `else` branch and hoping that it handles some of our cases), but I don't feel strongly enough to spend more time on it.  I'll delete the extra files and handle it the way you prefer.

I'll start on that (and other nits) now, and ping when it's ready for another check!